extras/ams_wrapper/src/api/types.py

```diff
@@ -74,16 +74,13 @@ def as_dict(self):
         }
 
 class SingleClassification:
-    def __init__(self, tag: Tag, attributes: List[Attribute] = None):
-        self.tag = tag
+    def __init__(self, attributes: List[Attribute] = None):
         self.attributes = attributes
 
     def as_dict(self):
-        result_dict = {
-            "tag": self.tag.as_dict(),
-        }
+        result_dict = {}
         if self.attributes is not None:
-            result_dict[attributes] = [attribute.as_dict() for attribute in self.attributes]
+            result_dict["attributes"] = [attribute.as_dict() for attribute in self.attributes]
```

Comment:
I believe these quotation marks are also required for entity type. Could you also fix it there?

Reply:
Done.